_posts/2020-07-28-10-Best-Practices-In-UX-UI-Design.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: post
+authors: [derya_duru]
+title: '10 Best Practices in UX and UI Design'
+image: /img/2020-07-28-Best-Practices-In-UX-UI-Design/design-banner.jpg
+tags: [Design, UX Design, UI Design]
+category: Design
+comments: true
+---
+
+> UX and UI design principles often lead to people thinking 'well duh!', until they actually have to try and create a user-friendly website or app. 
+> The basic gist of creating a user-friendly design is making sure the user doesn't have to think twice about doing something. 
+> The longer it takes for a user to be able to execute an action, the more annoyed they will get. 
+> The usability of your website or app can already be improved with just a few tweaks. Below is an overview of ten ways to make your users happy.
+
+# Table of contents
+* [Company logo placement and usage](#1-company-logo-placement-and-usage)
```

Comment:
Seeing as it is about a top 10, maybe use a numbered list instead of a bullet list?